aws/offsite/aurora-snapshots-fargate/package.json

```
@@ -4,7 +4,8 @@
   "description": "",
   "main": "restore-and-verify-backup.js",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "test-watch": "mocha --watch"
```

nit: prefer `:` to `-` for npm subcommands
```suggestion
    "test:watch": "mocha --watch"
```